code/game/objects/items/hfblade.dm

```diff
@@ -0,0 +1,142 @@
+// Chaplains begone!
+/obj/item/nullrod/scythe/vibro
+	name = "high fronquency bladd"
+
+/obj/item/melee/hfblade
+	name = "high frequency blade"
+	desc = "RULES OF NATURE"
+	icon_state = "hfblade"
+	item_state = "hfblade"
+	icon = 'icons/obj/hfblade.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	flags_1 = CONDUCT_1
+	obj_flags = UNIQUE_RENAME
+	force = 0.001
+	throwforce = 20
```

Comment:
Throw it and it does nothing, hmm

Reply:
Throwing it does almost the same damage as a fireaxe, jesus

Reply:
trowing it is not a really good idea considering how you just trew the only thing that gave you any edge in combat to the enemy to  pick it up and use it against you.